docs/.vuepress/components/metadata.vue

```diff
@@ -1,43 +1,117 @@
 <template>
   <header class="metadata" v-if="!isHomePage">
-    <span class="last-updated">
-      Last updated: <time :datatime="prettyDate">{{prettyDate}}</time>
+    <span v-if="updatedSinceLastVisit" class="last-updated last-updated--message">
```

Comment:
Is it possible to call `hasBeenUpdatedSinceLastVisit` from here directly? The extra `updatedSinceLastVisit` variable seems unnecessary.

The function can be pure, only return the result of `lastVisit < lastUpdate`.